packages/charts/react-charting/src/components/DeclarativeChart/helpers.ts

```diff
@@ -0,0 +1,280 @@
+import { create as d3Create, select as d3Select, Selection } from 'd3-selection';
+
+// const DOM_URL = window.URL || window.webkitURL;
+
+export interface IImageExportOptions {
+  width?: number;
+  height?: number;
+  scale?: number;
+  background?: string;
+}
+
+export function toImage(chartContainer?: HTMLElement | null, opts: IImageExportOptions = {}): Promise<string> {
+  return new Promise((resolve, reject) => {
+    if (!chartContainer) {
+      return reject(new Error('Chart container is not defined'));
+    }
+
+    try {
+      const background = opts.background || 'white';
```

Comment:
>'white' [](http://example.com/codeflow?start=44&length=7)

this wont work for dark mode and v9 theme where background color is a shade of grey

Reply:
This is only a failsafe. We are already setting the background based on the theme here: https://github.com/microsoft/fluentui/blob/539020103d513a6d61ff2d5121dbd5276ebc4916/packages/charts/react-charting/src/components/DeclarativeChart/DeclarativeChart.tsx#L112